uint32 version
uint32 totalPacketLen
uint32 platform
uint32 frameNumber
uint32 timeCpuCycles
uint32 numDetcetedObj
uint32 numTLVs
uint16 rangeBinIndexMax
uint16 rangeBinIndexPhase
float32 maxVal
uint32 processingCyclesOut
uint16 rangeBinStartIndex
uint16 rangeBinEndIndex
float32 unwrapPhasePeak_mm
float32 outputFilterBreathOut
float32 outputFilterHeartOut
float32 heartRateEst_FFT
float32 heartRateEst_FFT_4Hz
float32 heartRateEst_xCorr
float32 heartRateEst_peakCount
float32 breathingRateEst_FFT
float32 breathingRateEst_xCorr
float32 breathingRateEst_peakCount
float32 confidenceMetricBreathOut
float32 confidenceMetricBreathOut_xCorr
float32 confidenceMetricHeartOut
float32 confidenceMetricHeartOut_4Hz
float32 confidenceMetricHeartOut_xCorr
float32 sumEnergyBreathWfm
float32 sumEnergyHeartWfm
float32 motionDetectedFlag
float32 Reserved0
float32 Reserved1
float32 Reserved2
float32 Reserved3
float32 Reserved4
float32 Reserved5
float32 Reserved6
float32 Reserved7
float32 Reserved8
float32 Reserved9
